### ISSUE TITLE
[question]  same (vid, pid) in bootloader?

### ISSUE THREAD

**Issue by jindaxia** [xyzz/womier-flasher#1]
I have a Akko 3084 keyboard with the muc so called vs11k17A, as I  figure out that is SN32F268.

the (vid, pid) of this keyboard is (0x320F, 0x5013), after I use your program to reboot to the bootloader, the keyboard has no response and the leds also off. 

BUT , when i refresh the list , I will always see the  (0x320F, 0x5013) device, SO， IS it use the same  vid,pid In bootloader mode and the normal mode，

I repluggin the keyboard， the keyboard goes ok

sorry for my bad English

**Comment by xyzz** [comment]
Yes that is normal, if  you reboot through the software button it will not re-enumerate so it will keep the VID/PID.

If you boot to bootloader by shorting the boot pin, it will have the bootloader VID/PID.

regardless, you should not be using this software on unsupported keyboard unless you have original firmware backed up and also SWD access so that you can recover in case of brick.

**Comment by jindaxia** [comment]
Thanks
Another Question：
IS the womier K66‘s MCU  SN32F268？
In this branch， https://github.com/smp4488/qmk_firmware/tree/womier-k66-rgb-support 
/keybaords/womier/K66/boards/SN_SN32F260/board.h  
```
#define BOARD_GENERIC_SN32_F240B
#define BOARD_NAME "SN32F240B"

/*
 * MCU type as defined in the ST header.
 */
#define system_SN32F240B
```
how to understand this？
